insights/parsers/installed_rpms.py

```diff
@@ -369,8 +369,10 @@ def __init__(self, data):
         for k, v in data.items():
             setattr(self, k, v)
         self.epoch = data['epoch'] if 'epoch' in data and data['epoch'] != '(none)' else '0'
-        _gpg_key_pos = data.get('sigpgp', data.get('rsaheader', data.get('pgpsig_short', data.get('pgpsig', ''))))
-        self.redhat_signed = any(key in _gpg_key_pos for key in self.PRODUCT_SIGNING_KEYS)
+        _gpg_key_pos = data.get('sigpgp', data.get('rsaheader', data.get('pgpsig_short', data.get('pgpsig', data.get('vendor', '')))))
+        self.redhat_signed = "Unable_To_Determine"
```

Comment:
@xiangce - It is OK to me that return None when no necessary info in sosport, however, then we need to update rule in insights-plugins. @bfahr - For value for `self.redhat_signed` when no necessary in sosreport, what is your opinion?

Reply:
@wushiqinlou raising an exception is an option here since None is not an expected value type of the function.  Its up to you, as long as you document it in the function docstring.

**Opinion**
Generally if a function returns a value of type T, then a user of that function should be able to rely on getting a value of type T and not have to check for None. If the function is unable to return a value of type T then its appropriate for the function to raise an exception.  To me its the same as trying to access a dict key that is not there, you get an exception not None.

Whether the function returns None or an exception, the documentation must be clear so that the user of the function can code appropriately.

Reply:
@wushiqinlou I think None is better than the string, it is easier to check for assuming that rules are checking it properly. 
 If you do use a string then it should be defined as a attribute so that users, can use the attribute to compare instead of using a literal string that could be mistyped.